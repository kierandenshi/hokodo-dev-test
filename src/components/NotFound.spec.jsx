import React from 'react';
import { shallow } from 'enzyme';
import Component from './NotFound';

const defaultProps = {};

const getWrapper = (testProps = {}) => {
  const props = { ...defaultProps, ...testProps };
  return shallow(<Component {...props} />);
};

describe('NotFound component', () => {
  it('should match snapshot', () => {
    const wrapper = getWrapper();
    expect(wrapper).toMatchSnapshot();
  });
});
