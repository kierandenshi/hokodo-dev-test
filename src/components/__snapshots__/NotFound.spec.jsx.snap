// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should match snapshot 1`] = `
<div>
  No match 404
</div>
`;
